backend/src/index.js

```diff
@@ -2,11 +2,12 @@ const path = require('path');
 
 const express = require('express');
 const serveStatic = require('serve-static');
-const config = require('./config');
 
 const app = express();
 const server = require('http').Server(app);
 
+const config = require('./config');
```

Comment:
Est-ce que le nom est adéquat ?
```suggestion
const poney = require('./config');
```